README.md

```diff
@@ -349,16 +349,21 @@ for the actual, executable background task.
 
 # Testing
 
-It's possible to use Sqlite to emulate SQS when running the tests.
+It's possible to use SQLite to emulate SQS when running the tests.
 
 To do so, change the value of the env SQS_QUEUE_URL as the following examples:
 
 ```
-# to use a file
-SQS_QUEUE_URL=sqlite:/path/filename.sqlite3
+# To use a file
 
-# to use the memory
-SQS_QUEUE_URL=sqlite:/memory
+# It saves the data in /tmp/filename.sqlite3)
+SQS_QUEUE_URL=sqlite3://tmp/filename.sqlite3
+
+# It saves the data in /var/tmp/filename.sqlite3)
```

Comment:
Do we need two examples here? 🤔  Also, the trailing ")".

Reply:
It was a typo, I fixed that, tks!